Tests/Editor/PackagesTests.cs

```diff
@@ -13,19 +12,39 @@ class PackagesTests : TestFixtureBase
         [OneTimeSetUp]
         public void SetUp()
         {
-            AddRequest AddRequest = Client.Add("com.unity.2d.pixel-perfect@3.0.2");
-            while (AddRequest.Status != StatusCode.Success) {}
-            AddRequest = Client.Add("com.unity.services.vivox");
-            while (AddRequest.Status != StatusCode.Success) {}
+            AddRequest addRequest = Client.Add("com.unity.2d.pixel-perfect@3.0.2");
+            while (!addRequest.IsCompleted)
+                System.Threading.Thread.Sleep(10);
+            if (addRequest.Status == StatusCode.Failure)
+            {
+                throw new System.Exception("Could not install the required package (com.unity.2d.pixel-perfect@3.0.2). Make sure the package is able to be installed, and try again.");
```

Comment:
I'd rather use Assert's for these. Something like:
`Assert.True(addRequest.Status == StatusCode.Success, "Could not install the required package (com.unity.services.vivox). Make sure the package is able to be installed, and try again.");`